### PR TITLE
PAY-3830: Mobilepay and p24 added to payment methods

### PR DESCRIPTION
**Pull request by SvetaNesterenko**

### CODE REVIEW

**Review by themichaelyan — APPROVED** [review]

File: data/original/payment-methods.csv [hunk 1]
```diff
@@ -33,8 +33,10 @@ online,kcp_banktransfer,Bank Transfer via KCP,65,41,100,60,256,256,world,
 online,kcp_creditcard,Credit Card via KCP,65,41,100,60,256,256,world,
 online,kcp_payco,PayCo,65,41,100,60,256,256,world,
 online,klarna,Klarna,65,41,100,60,256,256,world,
+online,mobilepay,MobilePay,65,41,100,60,256,256,world,
 online,molpay_points,MOLPoints via MOLPay,65,41,100,60,256,256,world,
 online,multibanco,Multibanco,65,41,100,60,256,256,world,
+online,onlineBanking_PL,Przelewy24,65,41,100,60,256,256,world,
```
Comment (jlogeart) [inline data/original/payment-methods.csv:39]: Why this case?
Also, why this name and not `przelewy24`?

Comment (SvetaNesterenko) [reply data/original/payment-methods.csv:39]: it is defined as onlineBanking_PL in Adyen: 
https://docs.adyen.com/payment-methods/online-banking-poland/api-only/#make-payment

But I think we can change it to przelewy24 and use onlineBanking_PL internally when making a request to Adyen
What do you think?

Comment (jlogeart) [reply data/original/payment-methods.csv:39]: I agree - this is our reference, not Adyen's.

Did you check what other PSPs are saying?

Comment (SvetaNesterenko) [reply data/original/payment-methods.csv]: Renamed to przelewy24

**Review by ColmMathewsGlobalE — APPROVED** [review]

**Pull request merged**